src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php

```diff
@@ -73,6 +74,9 @@ public function GetHTML()
             $currentGroup = $mltRecord['group'];
             $connected = $mltRecord['connected'];
             $displayValue = $mltRecord['display_value'];
+
+            // TODO "display_value" can contain html (which is then rendered as text) - for example with gcf_CMSUserWithImage/callbackCmsUserWithImage for cms_user
```

Comment:
Therefore i would suggest using GetName here and not the display value, because the display value is meant for normal lists. 
But... what about Trees, that output a <ul> construct with the breadcrumb? Maybe we can`t encode the output here as HTML. On the other hand.... we did not change things here. So i would let it as before with encoding. 
Or move the output to a method so we can extend it if necessary for fields with this problem (never seen one yet).

Reply:
Changed it to GetName().

Did you see trees that output a checkbox matrix (as child elements)?

Reply:
No. Never. The only table with special HTML output in my mind is cms_user, but we don`t need an icon here. The name is enough. So GetName should be better in all cases.